src/components/feed/page/FeedUploadPage.tsx

```
@@ -57,15 +58,15 @@ export default function FeedUploadPage({ defaultValue, editingId, onSubmit }: Fe
     resetVote,
   } = useUploadFeedData(defaultValue);
 
-  const mobileContentsRef = useRef<HTMLTextAreaElement>(null);
+  const mobileContentsRef = useRef<HTMLDivElement>(null);
   const handleMobileKeyPressToContents = (e: React.KeyboardEvent<HTMLTextAreaElement>) => {
     if (e.key === 'Enter' && !e.nativeEvent.isComposing) {
       e.preventDefault();
       mobileContentsRef.current && mobileContentsRef.current.focus();
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)